src/auryn/SyncBuffer.h

```diff
@@ -58,6 +58,7 @@ namespace auryn {
 	{
 		private:
 			std::vector<NeuronID> send_buf;
+	                std::vector<NeuronID> send_bufx; // ALa added
```

Comment:
See also my comment above and my suggested implementation here: https://github.com/fzenke/auryn/tree/dev-syncbuffer